client/src/legacy/GridField.js

```diff
@@ -176,7 +176,8 @@ $.entwine('ss', function($) {
       this.find('.sortable-header').append('<th class="main col-Actions" />');
       this.find('tbody tr').each(function () {
         var cell = $(this).find('td:last');
-        cell.attr('colspan', 2);
+        var colspan = cell.attr("colspan") ?? 1;
+        cell.attr('colspan', colspan + 1);
```

Comment:
I assume the `+1` is what this is referring to from the PR description:

> This is required because this function adds an additional column to the table that needs to be filled in.

Can you please add a comment about that to the code? As is it's not immediately clear just by glancing at the code why we're doing this.

Reply:
It's not clear to me either why this still exists. It comes from the SS3 era, and by the look of it the reason for this fix is no longer relevant. Removing it altogether might be the way to go, but that comes with a larger impact. For now I have chosen to only fix the bug that this code causes to ensure that only the context of this bug is affected.